devs/docs/architecture.rst

```diff
@@ -0,0 +1,95 @@
+=====================
+CrateDB Architecture
+=====================
+
+This document assumes that you are familiar with using CrateDB. If you haven't
+used CrateDB before, please check out the [Getting
```

Comment:
The getting started link here is in markdown format, not in restructured text :wink:

Reply:
👍  fixed